getFeatureFlagById/main.go

```diff
@@ -6,37 +6,71 @@ import (
 	"net/http"
 
 	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/jwt"
+	"github.com/Real-Dev-Squad/feature-flag-backend/middleware"
 	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
 )
 
 func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
-	id, _ := req.PathParameters["flagId"]
+	corsResponse, err := middleware.CORSMiddleware()(req)
+	if err != nil {
+		log.Printf("CORS error: %v", err)
+		return corsResponse, err
+	}
+
+	if corsResponse.StatusCode != http.StatusOK {
+		return corsResponse, nil
+	}
```

Comment:
- should we move all the checks inside the coresMiddleware (or) return a function that can be called to perform all the checks required, to prevent duplicating this code everywhere `middleware.CORSMiddleware` is used?

Reply:
Sure